Add geometry creation from the relation editor widget (#4939)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/attribute_table.rst

* The |saveEdits| button is for saving all the edits.
* The |newTableRow| button will add a new record to the airport layer attribute table.
  And it will assign the new airport to the current region by default.

> * The |newTableRowWithGeom| is the same as |newTableRow| but let you digitize the

Reviewer: Why not 
```suggestion
* The |capturePoint| is the same as |newTableRow| but lets you digitize the
```
given that the substitution already exists? I'm afraid that this one gets cleared if someone runs the [find_set subst](https://github.com/qgis/QGIS-Documentation/blob/master/scripts/find_set_subst.py) script

>   airport geometry in the map canvas beforehand. Note that icon will change according

Reviewer: ```suggestion
  airport geometry in the map canvas beforehand. Note that the icon will change according
```

   :width: 1.5em
.. |newTableRow| image:: /static/common/mActionNewTableRow.png
   :width: 1.5em

> .. |newTableRowWithGeom| image:: /static/common/mActionCapturePoint.png

Reviewer: ```suggestion
.. |capturePoint| image:: /static/common/mActionCapturePoint.png
```
